sebs/sebs.py

```
@@ -20,37 +20,60 @@
+        """
+        filename_key = logging_filename if logging_filename else self.logging_filename
+        # Use a different key for None to avoid issues if filename could be ""
+        # For simplicity, assuming logging_filename being None means default log.
+        if filename_key in self._handlers:
+            return self._handlers[filename_key]
         else:
-            handlers = LoggingHandlers(verbose=self.verbose, filename=filename)
-            self._handlers[filename] = handlers
+            handlers = LoggingHandlers(verbose=self.verbose, filename=filename_key)
+            self._handlers[filename_key] = handlers
             return handlers
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)